momentum/4/modules/ha-proxy-client.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "ha_proxy_client - HAProxy protocol client module"
+description: "The ha_proxy_client module is used to configure Momentum to use HAProxy's PROXY protocol version 2 for outbound connections (see https://github.com/haproxy/haproxy/blob/master/doc/proxy-protocol.txt).  This can be leveraged in cases where your sending IPs are deployed on a different machine than Momentum.
+---
+
+## Configuration
+ 
+* When configured for a binding or binding group Momentum will connect to the given `ha_proxy_server` and prefix the SMTP session with a PROXY protocol version 2 header.  
+* The `dst_addr` and `dst_port` will be filled in with the resolved MX, the `src_addr` will be filled in with the configured value of `ha_proxy_src_addr` if the destination family is IPV4, or `ha_proxy_ipv6_src_addr` if the destination family is IPV6.  If you need to deliver to both IPV4 and IPV6 destinations than you must configure both options for the binding or binding_group.  
+* The `ha_proxy_bypass` option allows you to bypass the proxy and follow the normal delivery method on a domain by domain basis.
+* It is the customers responsibility to configure a listener at `ha_proxy_server` that   listens for PROXY protocol and forwards traffic based on `dst_addr:dst_port`. 
+
+A simple **example configuration** for HAProxy would be:
+
+```
+# listen to proxy protocol and forward to backend
+frontend  main
+    bind *:5000 accept-proxy
+    mode tcp
+    default_backend     out
+
+# forward to the destination using the clientip
+# from proxy protocol as the source address 
+backend out
+    mode tcp
+    source 0.0.0.0 usesrc clientip
+    server foo 0.0.0.0
+
+binding {
```

Comment:
I doubt anyone would be confused - but maybe put this in a separate markdown block as it's not the ha proxy conf -it's momentum config.

Reply:
I actually thought I did that already, but I guess I just did it in my head and not in the doc.

Reply:
You resolved this - but I still see it in the same code block?